src/main/scala/io/iohk/ethereum/network/handshaker/EthNodeStatus64ExchangeState.scala

```diff
@@ -15,8 +20,18 @@ case class EthNodeStatus64ExchangeState(
   import handshakerConfiguration._
 
   def applyResponseMessage: PartialFunction[Message, HandshakerState[PeerInfo]] = { case status: ETH64.Status =>
-    // TODO: validate fork id of the remote peer
-    applyRemoteStatusMessage(RemoteStatus(status))
+    import ForkIdValidator.syncIoLogger
+    (for {
+      validationResult <-
+        ForkIdValidator.validatePeer[SyncIO](blockchainReader.genesisHeader.hash, blockchainConfig)(
+          blockchainReader.getBestBlockNumber(),
+          status.forkId
+        )
+      _ = log.info("Fork id validation result was: {}", validationResult)
```

Comment:
do we want this logged on info level or debug would be sufficient?

Reply:
I think I'll remove this, as it should be logged by the `ForkIdValidator` on debug level